source/ingress_test.go

```diff
@@ -44,12 +44,11 @@ func (suite *IngressSuite) SetupTest() {
 	fakeClient := fake.NewSimpleClientset()
```

Comment:
In this file missing ipv4 with ipv6 test

Reply:
Since I am not changing the behaviour of the source or any code within, would this be in scope?